text/0000-dot-node-version-file-support.md

```diff
@@ -0,0 +1,63 @@
+- Feature Name: dot_node_version_file_support
+- Start Date: 2025-03-04
+- RFC PR: (leave this empty)
+- Volta Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Introduce support for a `.node-version` file, enabling it as a fallback source for determining project-specific Node versions. If node version is not specified in `package.json`, fallback to `.node-version` before using global defaults.
+
+# Motivation
+[motivation]: #motivation
+
+The aim is to align Volta with other popular Node version managers like fnm, which support `.node-version`. This enhances flexibility and user experience by allowing developers to specify their preferred Node versions in a widely recognized format.
+
+# Pedagogy
+[pedagogy]: #pedagogy
+
+Existing node developers are likely already familiar with other versioning methods like `.nvmrc`. Introducing support for `.node-version` aligns Volta with other node version managers.
+
+# Details
+[details]: #details
+
+## Inheritance from current solution
+
+The new feature will follow the same logic currently applied when reading a node version from `package.json`. This ensures consistency across different configuration files.
```

Comment:
3/5 (strong opinion: non-blocking)

> The new feature will follow the same logic currently applied when reading a node version from `package.json`.

I'm not 100% sure what this is referring to. It's probably best to define the current rules, instead of saying "follows the same logic currently applied".